web/src/app/monitors/monitor-create-edit/monitor-create-edit.component.html

```diff
@@ -0,0 +1,54 @@
+<div *ngIf="monitorForm" class="mat-content" fxLayout="column" fxLayoutGap="20px">
+  <form [formGroup]="monitorForm">
+    <mat-card>
+      <mat-card-header>
+        <mat-card-title> <span *ngIf="!isEdit">Create</span> <span *ngIf="isEdit"> Edit </span> Monitor</mat-card-title>
+      </mat-card-header>
+      <mat-divider></mat-divider>
+
+      <mat-card-content fxLayout="column" fxLayoutGap="20px">
+        <mat-form-field>
+          <input matInput type="text" placeholder="Name" formControlName="name" required>
+          <mat-error *ngIf="monitorForm.get(['name']).hasError('required')">This field is required.</mat-error>
+        </mat-form-field>
+        <mat-form-field>
+          <mat-select formControlName="method" placeholder="Method" required>
+            <mat-option value="get">Get</mat-option>
```

Comment:
The value probably should be capitals to match the model

Reply:
Which model ?

Reply:
The `MonitorModel` has `GET` and `POST` in capitals, the value here should match

Reply:
done